File: backend/HospitalFeedbackAPI/DTO/UserDto.cs
using HospitalFeedbackAPI.Models;

namespace HospitalFeedbackAPI.DTO
{
    public class UserDto
    {
        public required int Id { get; set; }
        public required string FullName { get; set; }
        public required string Email { get; set; }
        public required string Role { get; set; }
        public List<DoctorFeedback>? DoctorFeedbacks { get; set; }
        public List<HospitalFeedback>? HospitalFeedbacks { get; set; }
    }
}

File: mobile/app/src/main/java/com/enes/hospitalfeedbacksystem/model/UserDTO.kt
package com.enes.hospitalfeedbacksystem.model

data class UserDTO(val id : Int, val fullName : String,
                   val email: String, val role: String,
                   val doctorFeedbacks : List<DoctorFeedbackDTO>,
                   val hospitalFeedbacks : List<HospitalFeedbackDTO>)
